### PR TITLE
Improving steps for broker configuration

### PR DESCRIPTION
**Pull request by trwalke**
Improving steps for broker configuration.

### CODE REVIEW

File: 2-With-broker/README.md [hunk 1]
```diff
@@ -192,9 +193,19 @@ public override bool OpenUrl(UIApplication app, NSUrl url,
...
+{
+  base.OnActivityResult(requestCode, resultCode, data);
+  AuthenticationContinuationHelper.SetAuthenticationContinuationEventArgs(requestCode, resultCode, data);
+}
+```
+
 This method is invoked every time the application is launched and is used as an opportunity to process the response from the broker and complete the authentication process initiated by MSAL.NET.
 
-#### Step 3: Set a UIViewController()
+#### Step 3: Set a UIViewController() (iOS Only)
```
Comment (trwalke) [inline 2-With-broker/README.md:208]: It wasnt clear that certain steps were only for android or iOS

File: 2-With-broker/README.md [hunk 2]
```diff
@@ -415,17 +451,6 @@ Also, in order to make the token cache work and have the `AcquireTokenSilentAsyn
 1. In your project options, on iOS **Bundle Signing view**, select your `Entitlements.plist` file for the Custom Entitlements field.
 1. When signing a certificate, make sure XCode uses the same Apple Id.
 
-### Android specific considerations
```
Comment (trwalke) [inline 2-With-broker/README.md:418]: Moved to step 2

**Comment by trwalke** [comment]
@aiwangmicrosoft please make any changes you see fit

File: 2-With-broker/README.md [hunk 3]
```diff
@@ -309,10 +320,35 @@ NOTE: You also have the option of acquiring your redirect URI with code. see [Br
...
+<intent-filter>
+      <data android:scheme="msauth"
+                    android:host="com.microsoft.xforms.testApp"
+                    android:path="/hgbUYHVBYUTvuvT&Y6tr554365466="/>
+```
+**Please be sure to add a / in front of the signature in the "android:path" value**
+
+### Run the sample!
+
+If you configured you application to use the iOS broker, choose iOS as the platform you want to work on by setting the startup project in the Solution Explorer to the xamarin iOS project. Make sure that iOS is marked for build and deploy in the Configuration Manager. Alternatively, if you are using the Android broker you can follow the same steps to make Android the platform you choose to work on.
```
Comment (henrik-me) [inline 2-With-broker/README.md:350]: >you [](start = 18, length = 3)

your ?

Comment (henrik-me) [inline 2-With-broker/README.md:350]: >you [](start = 84, length = 3)

nit: consider removing you from this part of the sentence.

**Pull request merged**